auth-api/src/auth_api/services/affiliation.py

```diff
@@ -392,12 +393,23 @@ def fix_stale_affiliations(org_id: int, entity_details: Dict, environment: str =
 
         current_app.logger.debug('>fix_stale_affiliations')
 
+    @staticmethod
+    def _affiliation_details_url(affiliation: AffiliationModel) -> str:
```

Comment:
Had to move this out of the model, because of circular deps, can't call model -> service, but service -> service is fine.